compiler/wordrecg.nim

```diff
@@ -19,74 +19,302 @@ from strutils import cmpIgnoreStyle
-    wConst_cast, wDefault, wDelete, wDouble, wDynamic_cast,
-    wExplicit, wExtern, wFalse, wFloat, wFriend,
-    wGoto, wInt, wLong, wMutable, wNamespace, wNew, wOperator,
-    wPrivate, wProtected, wPublic, wRegister, wReinterpret_cast, wRestrict,
-    wShort, wSigned, wSizeof, wStatic_cast, wStruct, wSwitch,
-    wThis, wThrow, wTrue, wTypedef, wTypeid, wTypeof, wTypename,
-    wUnion, wPacked, wUnsigned, wVirtual, wVoid, wVolatile, wWchar_t,
-
-    wAlignas, wAlignof, wConstexpr, wDecltype, wNullptr, wNoexcept,
-    wThread_local, wStatic_assert, wChar16_t, wChar32_t,
-
-    wStdIn, wStdOut, wStdErr,
-
-    wInOut, wByCopy, wByRef, wOneWay,
-    wBitsize
+    wInvalid = ""
+
+    wAddr = "addr"
+    wAnd = "and"
+    wAs = "as"
+    wAsm = "asm"
+    wBind = "bind"
+    wBlock = "block"
+    wBreak = "break"
+    wCase = "case"
+    wCast = "cast"
+    wConcept = "concept"
+    wConst = "const"
+    wContinue = "continue"
+    wConverter = "converter"
+    wDefer = "defer"
+    wDiscard = "discard"
+    wDistinct = "distinct"
+    wDiv = "div"
+    wDo = "do"
+    wElif = "elif"
+    wElse = "else"
+    wEnd = "end"
+    wEnum = "enum"
+    wExcept = "except"
+    wExport = "export"
+    wFinally = "finally"
+    wFor = "for"
+    wFrom = "from"
+    wFunc = "func"
+    wIf = "if"
+    wImport = "import"
+    wIn = "in"
+    wInclude = "include"
+    wInterface = "interface"
+    wIs = "is"
+    wIsnot = "isnot"
+    wIterator = "iterator"
+    wLet = "let"
+    wMacro = "macro"
+    wMethod = "method"
+    wMixin = "mixin"
+    wMod = "mod"
+    wNil = "nil"
+    wNot = "not"
+    wNotin = "notin"
+    wObject = "object"
+    wOf = "of"
+    wOr = "or"
+    wOut = "out"
+    wProc = "proc"
+    wPtr = "ptr"
+    wRaise = "raise"
+    wRef = "ref"
+    wReturn = "return"
+    wShl = "shl"
+    wShr = "shr"
+    wStatic = "static"
+    wTemplate = "template"
+    wTry = "try"
+    wTuple = "tuple"
+    wType = "type"
+    wUsing = "using"
+    wVar = "var"
+    wWhen = "when"
+    wWhile = "while"
+    wXor = "xor"
+    wYield = "yield"
+
+    wColon = ":"
+    wColonColon = "::"
+    wEquals = "="
+    wDot = "."
+    wDotDot = ".."
+    wStar = "*"
+    wMinus = "-"
+    wMagic = "magic"
+    wThread = "thread"
+    wFinal = "final"
+    wProfiler = "profiler"
+    wMemTracker = "memtracker"
+    wObjChecks = "objchecks"
+    wIntDefine = "intdefine"
+    wStrDefine = "strdefine"
+    wBoolDefine = "booldefine"
+    wCursor = "cursor"
+
+    wImmediate = "immediate"
+    wConstructor = "constructor"
+    wDestructor = "destructor"
+    wDelegator = "delegator"
+    wOverride = "override"
+    wImportCpp = "importcpp"
+    wImportObjC = "importobjc"
+    wImportCompilerProc = "importCompilerProc"
+    wImportc = "importc"
+    wImportJs = "importjs"
+    wExportc = "exportc"
+    wExportCpp = "exportcpp"
+    wExportNims = "exportnims"
+    wIncompleteStruct = "incompleteStruct"
+    # deprecated
```

Comment:
ditto, indent

Reply:
Thanks! If I already do a big cleanup I might as well cleanup the junk that I leave around :D